cluster/manifests/prometheus/configmap.yaml

```diff
@@ -53,6 +53,17 @@ data:
       - record: job:skipper_average_latency:sum
         expr: sum(rate(skipper_serve_host_duration_seconds_sum{}[1m])) / sum(rate(skipper_serve_host_duration_seconds_count{}[1m]))
 
+    - name: skipper sum rate serve_host_duration by host,bucket
+      rules:
+      - record: job:skipper_serve_host_duration_host_bucket:sum
+        expr: sum(rate(skipper_serve_host_duration_seconds_bucket{}[1m])) by (le,host)
```

Comment:
promql question, isn't `sum(rate(skipper_serve_host_duration_seconds_bucket{}[1m])` == `skipper_serve_host_duration_seconds_bucket{}[1m]`

Reply:
I used the docs to create the query in the past, so I would say no.
`sum(rate(..` is the base of all histogram queries